app/models/phase.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: phases
+#
+#  id                     :bigint           not null, primary key
+#  uuid                   :uuid             not null, default: -> { "gen_random_uuid()" }
+#  title                  :string
+#  start_date             :datetime
+#  end_date               :datetime
+#  open_to_submissions     :boolean
+#  judging_criteria        :string
+#  judging_criteria_delta  :string
```

Comment:
more out of curiosity, do you know what the _delta fields are for, or why they're storing the length of the sring in a separate db column?

Reply:
These were used with the rich text editor on the challenge wizard. It was what was used to handle the conversion of the rich text to html.

This is what was used at the time https://quilljs.com/docs/delta

These might eventually be used on the rails app unless we find an alternate way to do the rich text